cfme/intelligence/chargeback/rates.py

```diff
@@ -133,8 +135,8 @@ def is_displayed(self):
             self.title.text == 'Storage Chargeback Rate "{}"'.format(self.obj.description))
 
 
-# TODO Inherit BaseEntity and create a parent collection class
-class ComputeRate(Updateable, Pretty, Navigatable):
+@attr.s
+class ComputeRate(Updateable, Pretty, Navigatable, BaseEntity):
```

Comment:
**Required**

`BaseEntity` provides navigation via `NavigatableMixin`, and `Navigatable` has to be removed from the parent classes here.